index.html

```diff
@@ -53,42 +53,235 @@ <h1>Am Alphy glad to see you!</h1>
+                <button>See Project</button>
+            </div>
+        </div>
+        <div class="project">
+            <div class="project-image">
+                <img src="./images/project_image2.png">
+            </div>
+            <div class="project-title">
+                <p>Multi-Post
+                    <br>Stories
+                </p>
+            </div>
+            <div class="projectsub-title">
+                <ul>
+                    <li class="one">Canopy</li>
+                    <li>Back end dev</li>
+                    <li>2015</li>
+                </ul>
+            </div>
+            <div class="project-description">
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            </div>
+            <div class="project-tag">
+                <ul>
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                </ul>
+            </div>
+            <div class="project-button">
+                <button>See Project</button>
+            </div>
+        </div>
+    
+    
+    <div class="project">
+        <div class="project-image">
+            <img src="./images/project_image3.png">
+        </div>
+        <div class="project-title">
+            <p>Tonic</p>
+        </div>
+        <div class="projectsub-title">
+            <ul>
+                <li class="one">Canopy</li>
+                <li>Back end dev</li>
+                <li>2015</li>
+            </ul>
+        </div>
+        <div class="project-description">
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        </div>
+        <div class="project-tag">
+            <ul>
+                <li>html</li>
+                <li>css</li>
+                <li>javascript</li>
+            </ul>
+        </div>
+        <div class="project-button">
+            <button>See Project</button>
+        </div>
+    </div>
+    <div class="project">
+        <div class="project-image">
+            <img src="./images/project_image4.png">
+        </div>
+        <div class="project-title">
+            <p>Multi-Post
+                <br>Stories
+            </p>
+        </div>
+        <div class="projectsub-title">
+            <ul>
+                <li class="one">Canopy</li>
+                <li>Back end dev</li>
+                <li>2015</li>
+            </ul>
+        </div>
+        <div class="project-description">
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        </div>
+        <div class="project-tag">
+            <ul>
+                <li>html</li>
+                <li>css</li>
+                <li>javascript</li>
+            </ul>
+        </div>
+        <div class="project-button">
+            <button>See Project</button>
+        </div>
+    </div>
+    </section>
+    <div class="about">
```

Comment:
- [ Required ] Good job working on the `About Myself` section it looks good, just one correction, I think it would be better if you changed the `<div class="about">` to the semantic `section` tag. Given that this is a logical section on your webpage the `section` tag would be the best tag to use.